hugo/content/changelog.md

```diff
@@ -6,6 +6,7 @@ lastmodifierdisplayname = "Naresh Rayapati"
 +++
```

Comment:
And also please update the docs at https://github.com/jenkinsci/jira-steps-plugin/tree/master/hugo/content/steps/comment folder appropriately. Thank you.